modules/wordads/wordads.php

```diff
@@ -491,6 +478,41 @@ function get_ad_snippet( $section_id, $height, $width, $location = '', $css = ''
 HTML;
 	}
 
+	/**
+	 * Returns the completed ad div with snipped to be inserted into the page
+	 *
+	 * @param  string  $spot top, side, inline, or belowpost
+	 * @param  string  $snippet The snippet to insert into the div
+	 * @param  array  $css_classes
+	 * @return string The supporting ad unit div
+	 *
+	 * @since 7.1
+	 */
+	function get_ad_div( $spot, $snippet, array $css_classes = array() ) {
+		if ( empty( $css_classes ) ) {
+			$css_classes = array();
+		}
```

Comment:
Not asking for a change, but genuinely curious. What would this catch? Misuse of the method?

Reply:
Just to make sure no one passes a null or something that otherwise wouldn't get blocked by the array check. I generally just like to ensure that I have a known good value as a default for a parameter :)